hibernate-core/src/main/java/org/hibernate/sql/results/graph/basic/BasicResultAssembler.java

```diff
@@ -58,7 +64,9 @@ public J assemble(
 			JdbcValuesSourceProcessingOptions options) {
 		final Object jdbcValue = extractRawValue( rowProcessingState );
 
-		ResultsLogger.RESULTS_MESSAGE_LOGGER.debugf( "Extracted JDBC value [%d] - [%s]", valuesArrayPosition, jdbcValue );
+		if ( RESULTS_LOGGER_DEBUG_ENABLED ) {
```

Comment:
```suggestion
		if ( ResultsLogger.DEBUG_ENABLED ) {
```

Reply:
Followed up in separate PR: https://github.com/hibernate/hibernate-orm/pull/6022